roi/roi_definer.py

```diff
@@ -0,0 +1,28 @@
+"""
+This is simply allows various programs to use the features found in the
+region of interest object
+"""
+
+
+class ROI:
+
+    def __init__(self, rowb, rowe, colb, cole):
+        self.rowb = rowb
+        self.rowe = rowe
+        self.colb = colb
+        self.cole = cole
+
+    def update(self, rowb, rowe, colb, cole):
+        if rowb >= 0 and rowe < 2048 and colb >= 0 and cole < 2048:
```

Comment:
let's use the pixel range of image you load in, instead of hard coding in number. So that this ``roi`` can be used widely